### PR TITLE
(WIP) feat(chore): zkSync node public release

### PR DESCRIPTION
**Pull request by githubdoramon**
# What :computer: 
* Renames External Node to zkSync node
* Adds doc changes of new features

# Why :hand:
* To improve clarity on what the node is (just a full node, no need to call it external)
* Improve docs on the new features/configs/ways of running

### CODE REVIEW

File: docs/infra/api-overview.md [hunk 1]
```diff
@@ -113,4 +113,4 @@ This namespace holds rollup-specific extensions to the Web3 API. Only the method
 
 ### `en` Namespace
 
-This namespace includes methods that external nodes call on the main node during syncing. If this namespace is active, other ENs can sync using this node.
+This namespace includes methods that zkSync nodes call on the main node during syncing. If this namespace is active, other kSync nodes can sync using this node.
```
Comment (MexicanAce) [inline docs/infra/api-overview.md:116]: typo
```suggestion
This namespace includes methods that zkSync nodes call on the main node during syncing. If this namespace is active, other zkSync nodes can sync using this node.
```

**Comment by dutterbutter** [comment]
@githubdoramon are you then adding the quickstart guide or is there still a need for me to do so?

**Comment by githubdoramon** [comment]
@dutterbutter I left the PR for Ago and Alex O to work on and do any additional changes. If you also have the context, please feel free to work on it :)

**Comment by agolajko** [comment]
@dutterbutter please add the guide!  https://github.com/matter-labs/zksync-era/blob/main/docs/guides/external-node/00_quick_start.md

Nothing to add apart from that, ready to push this to be public on the week of 13 May

**Comment by dutterbutter** [comment]
@agolajko added here: https://github.com/matter-labs/zksync-web-era-docs/pull/1063

**Comment by dutterbutter** [comment]
@agolajko you need to add `Gbps` to the spelling dictionary here: https://github.com/matter-labs/zksync-web-era-docs/blob/main/cspell-zksync.txt

**Review by dutterbutter — APPROVED** [review]

**Review by dutterbutter — APPROVED** [review]

**Pull request merged**